notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -620,7 +543,7 @@
         "id": "O-uCOL3Naap4"
       },
       "source": [
-        "Next, deploy your Vertex AI Model resource to a Vertex AI Endpoint resource. This operation will take 10-20 mins."
+        "Next, deploy your Vertex AI Model resource to a Vertex AI Endpoint resource. This operation  take 10-20 mins."
```

Comment:
- lower case "endpoint":
    - Vertex AI endpoint
- remove extra space and change "take" to "takes" and "mins" to "minutes"
    - This operation takes 10-20 minutes